src/components/Hero.jsx

```diff
@@ -0,0 +1,140 @@
+import styled from "styled-components";
+import BlueButton from "../generalComponents/BlueButtons";
+import GrayButton from "../generalComponents/GrayButton";
+import DisneyHero from '../image/DisneyHero.svg';
+import FedexHero from "../image/FedexHero.svg";
+import IntelHero from "../image/IntelHero.svg";
+import VisaHero from '../image/VisaHero.svg';
+import Carousel from "../image/Carousel.svg";
+import HeadingText from "../generalComponents/HeadingText";
+import SupportingText from "../generalComponents/SupportingText";
+
+const HeroComponent = styled.div`
+width: 1440px;
+height: 626px;
+background: #111928;
+display: flex;
+flex-direction: column;
+align-items: center;
+justify-content: space-around;
+`
+const HeroContainer = styled.div`
+width: 1280px;
+height: 434px;
+display: flex;
+flex-direction: row;
+justify-content: space-around;
+
+`
+const LeftContent = styled.div`
+width: 576px;
+height: 434px;
+gap: 32px;
+display: flex;
+flex-direction:column;
+`
+const LeftText = styled.div`
+width:576px;
+height: 206px;
+gap: 20px;
+`
+const LeftButtons = styled.div`
+width:380px;
+height:48px;
+gap: 16px;
+display: flex;
+`
+const Sponsors = styled.div`
+width: 576px;
+height: 116px;
+border: 1px 0px 0px 0px;
+padding: 32px 0px 0px 0px;
+gap: 12px;
+border-top: 1px solid #374151;
+display: flex;
+flex-direction: column;
+justify-content: space-around;
+`
+const SupportingTextSponsors = styled.div`
+width: 576px;
+height: 24px;
+font-family: Inter;
+font-weight: 500;
+font-size: 16px;
+line-height:24px;
+color: #6B7280;
+display:flex;
+`
+const Logos = styled.div`
+width:576px;
+height: 48px;
+display: flex;
+justify-content: space-between;
+`
+const IntelLogo = styled.svg`
```

Comment:
trzeba ujednolicić jakoś te style svg, co gdyby z backendu schodziło 200 logotypów? Każdy miałby własny komponent napisany ręcznie? Naprawdę ważne jest w aby kierować się zasadą responsywności i reużywalności, nawet gdyby oznaczało to odejście od wytycznych designerów :)